pset2/caesar/caesar.c

```diff
@@ -4,6 +4,17 @@
 #include <ctype.h>
 #include <cs50.h>
 
```

Comment:
//This code still looks a little cryptic, if you could make it a little more understandable. It would be great, I used the if else with islower and isupper arguments but this looks better.